### ISSUE TITLE
Template guidance on including graphics is not correct

### ISSUE THREAD

**Issue by bernhold** [betterscientificsoftware/betterscientificsoftware.github.io#198]
Compare https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/CuratedContent/IDEASSoftwareProductivityProject.md with http://bss.parallactic.com/resources/ideas-software-productivity-project

In GitHub UI, IDEAS logo shows up below the body of the item.  On Parallactic, the words "alt text" show instead.

If you look at the raw Markdown for the file, you'll see that the "alt text" is right there.  But the result really isn't want we want.  It would be better to display "IDEAS Logo" as the placeholder for the missing graphic or nothing at all.

I believe I constructed the Markdown from the template or from some other item that had a graphic in it.  Clearly that's not "the right answer".

Somewhat separately, I don't know whether we should expect the Parallactic site to be displaying graphics, so there may be additional issues there, or not.

There are probably a bunch of places this needs to be fixed, once we know what the right answer is.

**Comment by clararaubertas** [comment]
It says "alt text" because that's literally the text entered in the Markdown for the alt text (text displayed when a graphic is unavailable or a user's browser/screenreader doesn't support graphics)... if you change this to "IDEAS logo" you'll get the expected result.

Graphics should be displaying - let me know any issues.

**Comment by bernhold** [comment]
So we need to change the template guidance and update existing pages.

I now realize that the other text (with the image URL) is the hover text.

I do see graphics now.  Didn't earlier.  Thanks
